apps/web/src/components/AssessmentResultsTable/AssessmentResultsTable.tsx

```diff
@@ -0,0 +1,213 @@
+  };
+
+  const getColumns = () => {
+    // Sort the pools assessment steps then build columns for the poolCandidates assessment results
+    const sortedAssessmentSteps = assessmentSteps.sort((a, b) => {
+      if (a.sortOrder && b.sortOrder) {
+        return a.sortOrder - b.sortOrder;
+      }
+
+      if (a.title && b.title) {
+        return Intl.Collator(intl.locale).compare(
+          getLocalizedName(a.title, intl),
+          getLocalizedName(b.title, intl),
+        );
+      }
+
+      return 0;
+    });
+    const columns = sortedAssessmentSteps.reduce(
+      (
+        accumulator: ColumnDef<AssessmentTableRow>[],
+        assessmentStep: AssessmentStep,
+      ) => {
+        const type = assessmentStep.type ?? null;
+        const id =
+          getAssessmentStepType(type ?? "unknownType").id ??
+          uniqueId("results-table-column");
+        const status = columnStatus(assessmentStep, [
+          ...educationResults,
+          ...assessmentResults,
+        ]);
+        const header = columnHeader(
+          intl.formatMessage(getAssessmentStepType(type ?? "unknownType")),
+          status,
+          type,
+          intl,
+        );
+
+        return [
+          ...accumulator,
+          buildColumn({
+            id,
+            header,
+            poolCandidate,
+            assessmentStep,
+            intl,
+          }),
+        ];
+      },
+      [],
+    );
+
+    return [
+      // Insert skills header at top of columns order
+      columnHelper.accessor(
+        ({ poolSkill }) => getLocalizedName(poolSkill?.skill?.name, intl),
+        {
+          id: "requirement",
+          header: intl.formatMessage({
+            defaultMessage: "Requirement",
+            id: "jAWP0X",
+            description:
+              "Header for requirement section of assessment results table",
+          }),
+          cell: ({ row: { original } }) =>
+            cells.jsx(
+              <span>
+                {original.poolSkill ? (
+                  <>
+                    <span data-h2-font-weight="base(bold)">
+                      {getLocalizedName(original.poolSkill?.skill?.name, intl)}{" "}
+                    </span>
+                    <span>
+                      (
+                      {intl.formatMessage(
+                        original.poolSkill?.type
+                          ? getPoolSkillType(original.poolSkill.type)
+                          : commonMessages.notFound,
+                      )}
+                      )
+                    </span>
+                  </>
+                ) : (
+                  <span data-h2-font-weight="base(bold)">
+                    {intl.formatMessage({
+                      defaultMessage: "Education requirement",
+                      id: "4xXPIe",
+                      description: "Education requirement row header.",
+                    })}
+                  </span>
+                )}
+              </span>,
+            ),
+        },
```

Comment:
This is a style thing, so open to disagreement, but personally I think this would be a bit more readable if this is defined in a variable, like `const requirementsColumn` or `const firstColumn`, and then the overall function ends with 
```
return [
  requirementsColumn,
  ...columns
];
```
Right now the `...columns` is easy to miss at first glance :sweatsmile:

Reply:
Ya I like this style much better 👍 Fixed in [e143461](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9324/commits/e143461dafce737d41681066811503bb08a32d82)